src/pages/index.js

```diff
@@ -46,14 +106,26 @@ function handleCardClick(link, name) {
   imagePreviewModal.open({ name, link });
 }
 
+const modals = document.querySelectorAll(".modal");
+modals.forEach((modalElement) => {
+  const modalDelete = new Modal({ modalSelector: `#${modalElement.id}` });
+  modalDelete.setEventListeners();
```

Comment:
Please, delete it


You have already created instances of modals using the children of `Modal`. You can't use `Modal` in index.js. It's only for extending